rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -81,150 +83,293 @@ typedef struct
     RotationBatch rotation_batch;
 }JointsDataBatch;
 
+typedef struct MetaDataInfo {
+public:
+    uint _img_id = -1;
+    std::string _img_name = "";
+    ImgSize _img_size = {};
+} MetaDataInfo;
+
 struct MetaData
```

Comment:
@shobana-mcw Should we change these to classes or can we retain as struct?

Reply:
Not required